### DISCUSSION TITLE
Hi everyone. I'm Vasanth from india.

### DISCUSSION THREAD

**Discussion by Vasanthagokul** [gittogethers/octogatosconf22#119] [Where in the world?]
I'm just a fresher in the software world. I'm just looking for helpful advice to help me and my career.

**Comment by shyanukant** [comment]
Hey buddy, I'm also a beginner but I learned lots of technical things. 
### My advice is- 
1. Divide your learning into parts like frontend, backend, deployment, etc
    >**repeat step 1 - divide parts like fronted into ---HTML, CSS, JAVASCRIPT, FRAMEWORK**
1. Start with one, don't learn everything when you start, focus on that for around 1 month.
1. Start with easy then use it to make something(project) then go to the medium then make something and so on
    >**repeat steps 2 and 3 again and again**

And you don't need to purchase any premium course. there are lots of free resources available, search on google 
free platforms-
-  [Free Code Camp](https://www.freecodecamp.org/)
-  [Code With Harrry](https://www.codewithharry.com/)
-  [Community Classroom](https://www.commclassroom.org/)
